relays/ethereum/src/instances.rs

```diff
@@ -42,6 +42,22 @@ pub trait BridgeInstance: Send + Sync + std::fmt::Debug {
 	fn build_currency_exchange_call(&self, proof: Proof) -> Call;
 }
 
+/// Different instances supported by the relay.
+#[derive(Debug, Clone)]
+pub enum SupportedInstance {
+	Rialto,
+	Kovan,
+}
```

Comment:
When I introduced the trait objects for the supported instances we talked about not having an `enum`. However, due to ownership issues caused by the closure used for the retry mechanism I needed a way to create a new trait object during each retry, which lead me to introducing this `enum`.

It's not elegant, but I couldn't figure out another way to please the compiler.